src/app/manager-dashboard/manager-dashboard.component.html

```diff
@@ -35,9 +35,10 @@ <h3 i18n *ngIf="showParentList">{{ planetType === 'community' ? 'Nation' : 'Cent
     <p><b i18n>Local Version:</b> {{versionLocal}}</p>
     <ng-container [ngSwitch]="requestStatus">
       <ng-container *ngSwitchCase="'accepted'">
-        <a routerLink="resources" i18n mat-raised-button>List Resources</a>
-        <a routerLink="courses" i18n mat-raised-button>List Courses</a>
-        <a routerLink="meetups" i18n mat-raised-button>List Meetups</a>
+        <span><b i18n> List all from the {{ planetType === 'community' ? 'Nation' : 'Center' }}</b>
```

Comment:
We can use it similar to this
https://github.com/open-learning-exchange/planet/blob/e8b9a80afec2fbf945dc4fa0e10ae0cc952bf7a0/src/app/login/login.component.html#L8